### DISCUSSION TITLE
Soft delete method

### DISCUSSION THREAD

**Discussion by Neschastje** [Kamva/mgm#63] [Ideas]
Sometimes it is preferable to use soft delete with deleted_at field. It will be nice to have such mode in this library which changes Delete method(but still connected to hook) and adds deleted_at as null for collection filters.

**Comment by mehran-prs** [comment]
Hi @Kozyyy 
Thanks for your comment. Yes, it would be good to have this feature. 
I added #64 feature request and we will try to add this feature (soonly).
